circle.tex

```diff
@@ -2144,7 +2144,7 @@ \section{Connected \coverings over the circle}
   The Limited Principle of Omniscience (\cref{LPO})
   implies that the type of connected decidable \coverings over the circle is the sum
 of the component containing the universal \covering and for each positive integer $m$,
-the component containing the $m$-fold \covering.
+the component containing the $m$-fold \covering (\cref{def:mfoldS1cover}).
```

Comment:
This is the first mention of $m$-fold set bundles, perhaps the referenced definition should introduce this term (currently it uses the term _degree m function_).